lib/galaxy/model/__init__.py

```diff
@@ -2912,8 +2912,7 @@ def __init__(
     @property
     def active(self):
         if self.configured and not self.deleted:
-            # FIXME: don't included queued?
-            return not self.job.finished
+            return self.job.running
```

Comment:
This makes sense I think, I just wonder whether this should be its own PR?

Reply:
Happy to remove the last few commits then and add them as separate PR